.github/scripts/runner_determinator.py

```diff
@@ -72,7 +76,7 @@
 GITHUB_OUTPUT = os.getenv("GITHUB_OUTPUT", "")
 GH_OUTPUT_KEY_AMI = "runner-ami"
 GH_OUTPUT_KEY_LABEL_TYPE = "label-type"
-
+OPT_OUT_LABEL = "no-runner-determinator"
```

Comment:
nit: If we're disabling all the experiments, then let's label it accordingly

```suggestion
OPT_OUT_LABEL = "no-experiments"
```